cpp/data/zotero_cgi/qa.html

```diff
@@ -33,17 +44,17 @@ <h1>QA Settings</h1>
                 </tr>
             </thead>
             <tbody>
-                {LOOP tags, global_settings, journal_settings}
+                {LOOP tags, subfield_codes, regexes, record_types, global_settings, journal_settings}
```

Comment:
Maybe put rules for different record types into separate columns?

Reply:
We talked about this on the phone => i will change the table myself after the PR has been merged.